### ISSUE TITLE
glfwSwapInterval ignored when window is not visible

### ISSUE THREAD

**Issue by xanather** [glfw/glfw#891]
The call to glfwSwapInterval is ignored when the window is created with glfwWindowHint(GLFW_VISIBLE, GLFW_FALSE); and glfwSwapInterval is called before glfwShowWindow(m_window);

This is on windows, not sure about other platforms.

**Comment by elmindreda** [comment]
There is no global refresh rate; that is per monitor. A hidden window is not on any monitor. What should the OS do?

**Comment by xanather** [comment]
@elmindreda Yes thats right. My game loads everything and sets OpenGL states before the window is visible. A call to glfwSwapInterval() seems to be ignored when these startup operations are occurring, however once the window is visible and glfwSwapInterval() is called then vsync works as it should.

**Comment by elmindreda** [comment]
What GPU OEM is this and do you have Aero enabled?

**Comment by xanather** [comment]
OS: Windows 10  [10.0.0.14393] 64-bit
GPU: Nvidia GeForce GTX 570
I think Aero is enabled by default on windows 10?

**Comment by elmindreda** [comment]
Cannot reproduce on Windows 7 or Windows 10 with Nvidia cards.

**Comment by elmindreda** [comment]
It may be a driver issue. Not sure.

**Comment by tombsar** [comment]
@xanather Can you confirm that this is still an issue with the 3.3 release on updated Windows?
